.github/workflows/test.yml

```diff
@@ -15,3 +15,14 @@ jobs:
     - run: yarn install
     - run: yarn run build:test
     - run: yarn test
+  linux:
+    name: Unit test
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout sources
+      uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '14'
+    - run: yarn install
+    - run: yarn test:unit
```

Comment:
Why are you running this test twice?

Reply:
I misread `yarn build:test` for `yarn test:examples`. I'll remove the other test.

Reply:
Fixed